DigitalOffice/Tests/CheckRightsServiceTests/RepositoriesTests/CheckRightsRepositoryTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using CheckRightsService.Database;
+using CheckRightsService.Database.Entities;
+using CheckRightsService.Mappers;
+using CheckRightsService.Mappers.Interfaces;
+using CheckRightsService.Models;
+using CheckRightsService.Repositories;
+using CheckRightsService.Repositories.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+
+namespace CheckRightsServiceUnitTests.RepositoriesTests
+{
+    class CheckRightsRepositoryTests
+    {
+        private ICheckRightsRepository repository;
+        private DbContextOptions<CheckRightsServiceDbContext> dbOptionsGetRightsList;
+        private CheckRightsServiceDbContext dbContext;
+        private IMapper<DbRight, Right> mapper;
+        [SetUp]
+        public void Setup()
+        {
+            mapper = new RightsMapper();
+            dbOptionsGetRightsList = new DbContextOptionsBuilder<CheckRightsServiceDbContext>()
+                                    .UseInMemoryDatabase(databaseName: "InMemoryDatabase")
+                                    .Options;
+            dbContext = new CheckRightsServiceDbContext(dbOptionsGetRightsList);
```

Comment:
Put something in dbContext and check that repository return something.

Reply:
fixed